crowbar_framework/app/models/service_object.rb

```diff
@@ -1529,8 +1536,6 @@ def run_remote_chef_client(node, command, logfile_name)
       ssh = ["sudo", "-u", "root", "--", "ssh", "-o", "TCPKeepAlive=no", "-o", "ServerAliveInterval=15", "root@#{node}"]
       ssh_cmd = ssh.dup << command
 
-      # check if there are currently other chef-client runs on the node
-      wait_for_chef_clients(node)
```

Comment:
Thinking out loud: I wonder if we should keep this here, just in case there's a manual chef-client (or the chef-client daemon got restarted). Of course, this means we should change `wait_for_chef_clients` to accept running with and without logger.

What do you think?

(other than that, I'm +1)